{
  "id": 5600845,
  "key": "MM-T2696",
  "name": "Invalid server name",
  "createdOn": "2020-06-01T17:46:17Z",
  "objective": null,
  "precondition": null,
  "estimatedTime": null,
  "labels": ["se-prod", "ldap"],
  "component": null,
  "priority": { "id": 401951, "name": "Low" },
  "status": { "id": 890281, "name": "Active" },
  "folder": {
    "id": 404269,
    "name": "AD LDAP",
    "parentId": 2954447,
    "fullNames": ["Suite", "AD LDAP"],
    "fullPath": "suite/ad-ldap"
  },
  "customFields": {
    "Playwright": null,
    "Authors": "",
    "Priority P1 to P4": "P4 - Low-Impact (Edge or unsuitable to repeat?)",
    "Update notes": "",
    "Rainforest": ["N/A"],
    "Team Ownership": ["Suite Users"],
    "Manual Test Environments": [],
    "Detox": null,
    "Cypress": "To Do",
    "MMCTL": null,
    "Last Updated": "",
    "Tags": [],
    "Location": "AD LDAP"
  },
  "steps": [
    {
      "inline": {
        "description": "Invalid server name<br>————————————————————————————<ol><li>Load AD/LDAP page in system console, with all the settings filled in</li><li>Delete the last letter of the AD/LDAP server field so it reads for example `corp.mattermost.co`, and Save</li><li>At the bottom of the page, click \"AD/LDAP Test\"</li></ol>",
        "testData": null,
        "expectedResult": "<ul><li>Error message saying the connection was not successful.</li><li>Set back to correct setting such as</li><li>corp.mattermost.com and test again (successful)</li></ul>",
        "customFields": {
          "Fix Versions": [],
          "Related ticket(s)": "",
          "Important notes about this step": ""
        }
      },
      "testCase": null
    }
  ],
  "caseHashed": "27bb8634a1b150db7cafbf7ea25e3c833e23d5011694aa8227c777865ad24ced3dff51ab89760c29ce6e621d696a471b",
  "stepsHashed": "5d66b0ee64f1805877cada6cedf006ae8364c2aa63996de404d1c59ee7bac710c86df7e9617608cdce1317d1165f417d"
}
